erizoAPI/MediaStream.cc

```diff
@@ -56,7 +56,7 @@ MediaStream::MediaStream() : closed_{false}, id_{"undefined"} {
   future_async_ = new uv_async_t;
   uv_async_init(uv_default_loop(), async_stats_, &MediaStream::statsCallback);
   uv_async_init(uv_default_loop(), async_event_, &MediaStream::eventCallback);
-  uv_async_init(uv_default_loop(), future_async_, &MediaStream::promiseResolver);
+  uv_async_init(uv_default_loop(), future_async_, &MediaStream::closePromiseResolver);
```

Comment:
np: shouldn't we call this `close_future_async_`?

Reply:
Sure 👍